BusinessTampereTrafficMonitoring/traffic_lights/signal_group.py

```diff
@@ -12,6 +22,15 @@ def __init__(self, device: str, name: str, timestamp: str, status_code: str):
             self.t_red_start = timestamp
 
     def __create_event(self, timestamp):
+        """
+
```

Comment:
Try to be consistent with the empty lines.  There shouldn't be an empty line after the opening `"""` of a docstring.